src/controllers/webhooks.js

```diff
@@ -16,11 +15,11 @@ function findById(orderNumber) {
 }
 
 function incrementCacheScore(teamID, teamName, quantity) {
-  incrementTeamScore(teamID, teamName, quantity);
+  models.Team.increment(['itemsSold'], { by: quantity, where: { teamId: teamID } });
```

Comment:
should `teamId` just be `id`? same for below

Also, I would consider renaming this function or just deleting it altogether and doing the db mutation inline (like how the orders table is being mutated inline),

Reply:
Is the attribute in the team model not called`TeamId`, not sure if it should be `id`. Will rename function instead of deleting for readability.